lab_1/main.py

```diff
@@ -4,6 +4,20 @@
 """
 
 
+def read_from_file(path_to_file: str) -> str:
+    """
+    Opens the file and reads its content
+    :return: the initial text in string format
+    """
+    with open(path_to_file, 'r', encoding='utf-8') as fs:
+        data = fs.read()
+
+    return data
+
+
+text = read_from_file('data.txt')
```

Comment:
There should be no function calling\using in `main.py` -> use functions on the `start.py`.